stop-mycroft.sh

```diff
@@ -40,37 +41,37 @@ function help() {
     exit 0
 }
 
-function process-running() {
-    if [[ $( pgrep -f "python3 (.*)-m mycroft.*${1}" ) ]] ; then
+process_running() {
+    if [ "$( pgrep -f "python3 (.*)-m mycroft.*${1}" )" ] ; then
         return 0
     else
         return 1
     fi
 }
 
-function end-process() {
-    if process-running $1 ; then
+end_process() {
+    if process_running "$1" ; then
         # Find the process by name, only returning the oldest if it has children
         pid=$( pgrep -o -f "python3 (.*)-m mycroft.*${1}" )
-        echo -n "Stopping $1 (${pid})..."
-        kill -SIGINT ${pid}
+        printf "Stopping %s (%s)..." "$1" "${pid}"
+        kill -SIGINT "${pid}"
```

Comment:
Was looking at this to see if it could be merged got an issue with this:

```
Stopping skills (6297)..../stop-mycroft.sh: 57: kill: Illegal option -S
failed to stop.
  Killing skills (6297)...
killed.
Stopping audio (6300)..../stop-mycroft.sh: 57: kill: Illegal option -S
failed to stop.
  Killing audio (6300)...
killed.
Stopping speech (6303)..../stop-mycroft.sh: 57: kill: Illegal option -S
failed to stop.
  Killing speech (6303)...
killed.
Stopping enclosure (6306)..../stop-mycroft.sh: 57: kill: Illegal option -S
failed to stop.
  Killing enclosure (6306)...
killed.
Stopping messagebus.service (17057)..../stop-mycroft.sh: 57: kill: Illegal option -S
failed to stop.
  Killing messagebus.service (17057)...
killed.
```

seems like kill in sh doesn't work with signal names?

Reply:
Is `-SIGINT` equivelant to `-s SIGINT`? I believe sh just requires the `-s` option before specifying what signal to send. I just pushed a change, could you try again?

Reply:
Now it's `Stopping skills (5419)..../stop-mycroft.sh: 57: kill: invalid signal number or name: SIGINT` `kill -l` in sh suggests it should be just `INT`

Reply:
How strange, fixed.

Reply:
Yes, sh seems to be Dash on my machine.

Reply:
Working now. :+1: